docs/sources/flow/get-started/install/ansible.md

```diff
@@ -0,0 +1,87 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/get-started/install/ansible/
+canonical: https://grafana.com/docs/agent/latest/flow/get-started/install/ansible/
+description: Learn how to install Grafana Agent Flow with Ansible
+menuTitle: Ansible
+title: Install Grafana Agent Flow with Ansible
+weight: 550
+---
+
+# Install or uninstall {{% param "PRODUCT_NAME" %}} using Ansible
+
+If your machines are already managed with Ansible, you can install {{< param "PRODUCT_NAME" >}} from
+our package repositories with these steps.
+
+## Before you begin
+
+- These steps assume you already have a working [Ansible](https://www.ansible.com/) setup,
+and a pre-existing inventory.
+- You can add the tasks below to any new or existing Role you choose.
+- These tasks install from Grafana's package repositories. They expect to target linux systems using 
+apt or yum package managers.
+
+## Steps
+
+To add {{% param "PRODUCT_NAME" %}} to a host:
+
+1. Add these tasks to your playbook to add the Grafana package repositories to your system:
+    ```yaml
+    - name: "Install deb repo"
+      when:
+        - "ansible_pkg_mgr == 'apt'"
+      block:
+        - name: "Import Grafana apt gpg key"
+          ansible.builtin.get_url:
+            url: "https://apt.grafana.com/gpg.key"
+            dest: /usr/share/keyrings/grafana.key
+            mode: "0644"
+
+        - name: "Add Grafana apt repository"
+          ansible.builtin.apt_repository:
+            repo: "deb [signed-by=/usr/share/keyrings/grafana.key] https://apt.grafana.com/ stable main"
+            state: present
+            update_cache: true
+    - name: "Install yum repo"
+      when:
+        - "ansible_pkg_mgr in ['yum', 'dnf']"
+      block:
+        - name: "Add Grafana yum/dnf repository"
+          ansible.builtin.yum_repository:
+            name: grafana
+            description: grafana
+            baseurl: "https://packages.grafana.com/oss/rpm"
+            enabled: true
+            gpgkey: "https://packages.grafana.com/gpg.key"
```

Comment:
```suggestion
            baseurl: "https://rpm.grafana.com/oss/rpm"
            enabled: true
            gpgkey: "https://rpm.grafana.com/gpg.key"
```

The info documented here: https://packages.grafana.com/ shows `baseurl=https://rpm.grafana.com`  Does Ansible work with the deeper dir structure or does it need/work with the root URL?

Same path documented for `gpgkey`  Just changing for consistency in all sources... https://packages.grafana.com/ and https://rpm.grafana.com/ and https://grafana.com/docs/agent/latest/flow/setup/install/linux/